lib/screens/v2/widget_instance/subway_status.ex

```diff
@@ -13,7 +13,7 @@ defmodule Screens.V2.WidgetInstance.SubwayStatus do
 
   @type t :: %__MODULE__{
           screen: Screen.t(),
-          subway_alerts: list(Alert.t())
+          subway_alerts: list(%{alert: Alert.t(), context: %{}})
```

Comment:
Could this be a struct rather than a map? This would give us better compile-time checks and would make it clearer what the overall data structure "is"/represents.